src/restLevelClient/generateClientDefinition.ts

```diff
@@ -31,6 +31,7 @@ import {
 } from "./generateMethodShortcuts";
 import { Methods, PathParameter, Paths } from "./interfaces";
 export let pathDictionary: Paths = {};
+let requireHttpResponse = false;
```

Comment:
I don't really love the idea of having these global variables. These should be within the scope of the function that is using them.

One option is to pass an array `const coreClientImports = new Set(['Client'])` to `getOperationReturnType` the same way wo do with `importedModels`.

Having global variables is generally a bad practice and should be avoided when possible

Reply:
ok, Let me make the change